rmgpy/statmech/conformerTest.py

```diff
@@ -226,7 +233,8 @@ def test_getTotalMass(self):
         """
         Test the Conformer.getTotalMass() method.
         """
-        self.assertAlmostEqual(self.conformer.getTotalMass()*constants.Na*1000., numpy.sum(self.mass), 6)
+        self.assertAlmostEqual(self.conformer.getTotalMass() *
+                               constants.Na*1000., numpy.sum(self.mass), 6)
```

Comment:
I think additional spaces around `*` are required

Reply:
I think it would also be preferable to split after the comma instead of in the middle of the expression.

Reply:
Modified.